src/screens/Button.js

```diff
@@ -1022,6 +1022,33 @@ const ButtonPage = () => (
           </PropertyValue>
         </Property>
 
+        <Property name="button.icon.size">
+          <Description>
+            Adjustments to the icon size on different screen sizes.
+          </Description>
+          <PropertyValue type="object">
+            <Example>{`{
+              small: '12px',
+              medium: '18px',
+              large: '28px',
+            }`}</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="button.[kind].direction">
```

Comment:
It looks like direction could also be used within option so `button.option.direction`. Also I'm not sure if it will be confusing to have `[kind]` just because for the other props in the docs we list them out for the different kinds

Reply:
yeah I wasnt sure either I can list them out how we have them but thought we could simplify doing this way. Ill list them out though